src/swagger/README.md

```diff
@@ -10,14 +10,16 @@ our OpenAPI definition, and [RapiDoc](https://mrin9.github.io/RapiDoc/index.html
 In it's current state our OpenAPI documentation is _almost_ an accurate representation of the actual behavior of the API. The first version of these new docs was made based on models and responses defined in the existing documentation. A number of small inconsistencies were discovered that have not been addressed.
 
 ### TODO
-- [ ] document query parameters
+- [x] document query parameters
 - [ ] standardize schema object naming (mostly cleanup the /schemas directory)
 - [ ] validate schemas against models or actual api responses
 - [ ] validate schema and field descriptions are accurate
 - [ ] reorganize tag ordering
 - [ ] add tag descriptions
 - [ ] add section in overview with summary of SRD / OGL
```

Comment:
Did you want to keep any of these?

Reply:
i think these are all still applicable / things that would be useful. the one i forgot to check off was the last one because i did add some info regarding SRD/OGL (more or less copy/pastes of your posts in the FAQ channel in the discord :) )

Reply:
Oh weird. Sorry. For some reason my brain parsed them as being removed.